setup.py

```diff
@@ -12,7 +12,7 @@
     install_requires=[
         "attrdict>=2.0.1",
         "requests>=2.23.0",
-        "websocket_client>=0.57.0",
+        "websocket-client>=0.57.0",
```

Comment:
Heh, so apparently websocket-client or websocket_client should work. So, if websocket-client is working with conda for you but websocket_client isn't. I'm fine with leaving this.